languages/ruby/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,101 @@
+There are two different types of numbers in Ruby:
+
+- Integers: numbers with no digits behind the decimal separator (whole numbers). Examples are `-6`, `0`, `1`, `25`, `976` and `500000`.
+- Floating-point numbers: numbers with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+They are implemented through the [`Integer`][integer-ruby] and [`Float`][float-ruby] class.
+Arithmetic is done using the standard arithmetic operators. Numbers can be compared using the standard numeric comparison operators(`>`, `<=`, `==`, `!=`, etc.).
+Basic arithmetic operations between instances of `Integer`, will always result in an instance of `Integer`:
+
+###`+`
```

Comment:
One thing to note is that students that will be working on this exercise, must have completed the `basics` exercise (speaking of which, you should update Ruby's [`config.json` file](https://github.com/exercism/v3/blob/master/languages/ruby/config.json)). As such, they'll be familiar with some basic arithmetic operators like `*` and `+` (see the [basics example](https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/basics/.meta/example.rb) and the [basics after.md](https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/basics/.docs/after.md)). Could you take a look at which things you can assume a student will already know and remove what the student already knows?

Another thing to note is that the `introduction.md` file should contain just the minimal amount of information for a student to solve it. Maybe you could take a look at whether everything in the introduction is required for the student to know in order to complete the exercise? If it is not required, it should be removed from the introduction. Note that the `after.md` allows for a more expansive description of the concept, and can also contain non-essential knowledge.